include/pluginplay/module_manager.hpp

```diff
@@ -231,6 +231,14 @@ class ModuleManager {
      */
     runtime_type& get_runtime() const noexcept;
 
+    /** @brief Returns the keys of all the modules in the module manager.
+     *
+     * @return A vector of keys for all the modules in the module manager.
+     *
+     * @throw None No throw guarantee.
+     */
+    std::vector<type::key> keys() const noexcept;
```

Comment:
Similar to the rest of the API, please define a typedef for the return type and use the typedef (ensures the type stays consistent throughout the API). Maybe something like `key_container`? Also, are you sure this is no throw guarantee?

Reply:
Done with [63c6de0](https://github.com/NWChemEx-Project/PluginPlay/pull/326/commits/63c6de0a9bb6265b2143d3befaa7487140da8b3b).